logstash-core/lib/logstash/agent.rb

```diff
@@ -22,7 +22,7 @@ class LogStash::Agent
 
   def initialize(options = {})
     @pipelines = {}
-     
+
```

Comment:
always found this changes very relevant, no just kidding sometimes git thinks we deleted and added a new line :panda_face: